spec/data/lwrp/providers/embedded_resource_accesses_providers_scope.rb

```diff
@@ -24,5 +24,5 @@ def action_twiddle_thumbs
 end
 
 def generate_new_name(str, &block)
-  "#{str}, #{block.call}"
+  "#{str}, #{yield}"
```

Comment:
If you're not using the `block` parameter they can be removed from the method's signature.

Reply:
Let me revertthis. I was trying to stick to just the match? fixes in this PR